core/unit_test/TestViewLayoutStrideAssignment.hpp

```diff
@@ -617,6 +617,171 @@ TEST(TEST_CATEGORY, view_layoutstride_right_to_layoutright_assignment) {
   }
 }
 
+TEST(TEST_CATEGORY, view_layoutstride_right_to_layoutleft_assignable) {
+  typedef TEST_EXECSPACE exec_space;
+
+  auto t = time(nullptr);
+  srand(t);  // Use current time as seed for random generator
```

Comment:
**Please** never *ever* use time-seeded random numbers (or any other non-deterministic functionality) in a unit test. (Also, I don't see any reason to use random numbers here in the first place.)

Reply:
This is just a copy of some over test (or it was there before I guess) but still we probably should not do that for tests where this isn't absolutely necessary (which mostly is probably just the Random Number Generator tests).

Reply:
So are we going to remove this before we merge?